### PR TITLE
Bump kube-prometheus-stack chart version

### LINKED ISSUE #3279: Bump Dependencies for MetalK8s 2.10
**Component**:

'build'

**Why this is needed**:

We didn't update dependencies such as Prometheus, Grafana, etc ... since branch 2.8. We want to always embed the latest fix, security fixes, or new features and do it before releasing the first patch version of a specific branch.

**What should be done**:

Check and update all dependencies and make sure we don't break anything ...

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: charts, salt, build

**Context**: 

**Summary**:
Bump kube-prometheus-stack chart version to the latest

**Acceptance criteria**: 

---

See: #3279

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,18 @@
 - Bump CoreDNS version to 1.8.0
   (PR[#3354](https://github.com/scality/metalk8s/pull/3354))
 
+- [#3279](https://github.com/scality/metalk8s/issues/3279) - Bump
+  kube-prometheus-stack charts version from 12.2.3 to 15.4.4.
+  The following images have also been bumped accordingly:
+   - grafana from 7.3.5 to 7.5.5
+   - k8s-sidecar from 1.1.0 to 1.10.7
+   - kube-state-metrics from v1.9.7 to v1.9.8
+   - node-exporter from v1.1.2 to v1.12
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: typo 
```suggestion
   - node-exporter from v1.1.2 to v1.1.2
```

File: charts/kube-prometheus-stack.yaml [hunk 2]
```diff
@@ -181,11 +172,11 @@ grafana:
       nginx.ingress.kubernetes.io/rewrite-target: /$2
     path: /grafana(/|$)(.*)
     hosts:
-      - null
+      - 'null'
 
   grafana.ini:
     server:
-      root_url: '__escape__(https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana)'
+      root_url: '__escape__(https://{{ "{{ grains.metalk8s.control_plane_ip }}" }}:8443/grafana)'
```
Comment (TeddyAndrieux) [inline charts/kube-prometheus-stack.yaml:179]: Why it's needed ?

**Review by TeddyAndrieux — DISMISSED** [review]
Honestly impossible to review but since it seems to works ok for me

File: charts/kube-prometheus-stack.yaml (hunk 2 above)
Comment (alexandre-allard) [reply charts/kube-prometheus-stack.yaml:179]: It's in the commit message https://github.com/scality/metalk8s/pull/3360/commits/959c271f9f4d09ecc1a8fc3e61c02c1d793212e8 :)
Basically, Helm templating is using the same tags as Jinja and it conflicts, so we need to escape them.

File: CHANGELOG.md (hunk 1 above)
Comment (alexandre-allard) [reply CHANGELOG.md]: Good catch, I'm too far in the future ! :p (actually it's even 1.0.1 to 1.1.2...)

File: charts/kube-prometheus-stack.yaml (hunk 2 above)
Comment (TeddyAndrieux) [reply charts/kube-prometheus-stack.yaml:179]: Oh !  Right :+1:

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**